beacon-chain/sync/service.go

```diff
@@ -277,6 +277,11 @@ func (s *Service) registerHandlers() {
 	}
 }
 
+// marks the chain as having started.
+func (s *Service) markForChainStart() {
+	s.chainStarted = true
```

Comment:
Do we want to add a lock ?

Reply:
This method can only be called once and that too from a single routine. I don't think a lock is necessary here